test/integration/builder/transaction.js

```diff
@@ -397,8 +401,8 @@ module.exports = function(knex) {
       if (/mssql/i.test(knex.client.driverName)) {
         return Promise.resolve();
       }
-      const first = Bluebird.delay(50);
-      const second = first.then(() => Bluebird.delay(50));
+      const first = delay(50);
+      const second = first.then(() => delay(50));
```

Comment:
Your fix here looks good, but this is another test case we should revisit later on.

Looking at the code, I'm not convinced that this test case is sufficient / reliable.  (Especially since we recently discovered that some `Transaction` subclasses were not handling this case properly).  So, we'll need to take another look at this once we finish w/ the Bluebird removal.